### ISSUE TITLE
Redstone Kicks me out of my server.

### ISSUE THREAD

**Issue by shadowknight484** [TheLimePixel/GregBlock#260]
So i was playing gregblock 2.2.1 on my server on aternos and i place redstone and then it crashes my game. I can turn the server on but when i try and join it just crashes my game.

**Comment by sirmewsalotthe3rd** [comment]
yes, putting Redstone on top of most GTCE block will corrupt the entire world and you should replace your world with one of the backups that are automatically made
